src/main/java/com/rpi/alexandria/controller/RatingController.java

```diff
@@ -0,0 +1,192 @@
+package com.rpi.alexandria.controller;
+
+import com.nimbusds.jose.shaded.json.JSONObject;
+import com.rpi.alexandria.controller.response.AppResponse;
+import com.rpi.alexandria.dto.RatingDTO;
+import com.rpi.alexandria.model.Rating;
+import com.rpi.alexandria.model.User;
+import com.rpi.alexandria.service.UserService;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.OffsetDateTime;
+import java.util.*;
+
+@RestController
+@RequestMapping("/api/rating")
+@Slf4j
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+@CrossOrigin("http://localhost:${ui.port}")
+public class RatingController extends BaseController {
+
+	public RatingController(UserService userService) {
+		super(userService);
+	}
+
+	@PostMapping("/add")
```

Comment:
We're following RESTful architecture. So the APIs need to follow the RESTful practices. Change it to:
`@PostMapping("/")`

Reply:
updated